README.md

```diff
@@ -190,6 +194,7 @@ my-application-alb:
         - internal.my-web-service-2
       enable_cdn_record: false
       enable_logging: true
+      cdn_timeout_seconds: 60
```

Comment:
I can't decide whether I like that fact that we make it obvious it's seconds, or would prefer just `cdn_timeout`.  E.g. we don't have `enable_logging_boolean`.

I wonder what other people think.

Reply:
I think we're stuck with it. I had to do similar [here](https://github.com/uktrade/terraform-platform-modules/pull/254/files#diff-be639a729ee265634f6b35550c54684eb315c7e7c1306efb6ca0fe1695664e28R427-R435) and it's a disgusting syntax.

Reply:
> I can't decide whether I like that fact that we make it obvious it's seconds, or would prefer just `cdn_timeout`. E.g. we don't have `enable_logging_boolean`.
> 
> I wonder what other people think.

That's the name as it was defined in the ticket. I'm partial to both sides, though I am erring more towards retaining `seconds` in the name just because I've seen timeouts defined in minutes before

Reply:
milliseconds is common for timeouts so I think this is good

Reply:
I am persuaded.